dftimewolf/lib/collectors/filesystem.py

```diff
@@ -39,13 +39,18 @@ def SetUp(self, paths=None):  # pylint: disable=arguments-differ
 
   def Process(self):
     """Collects paths from the local file system."""
+    file_added = False
     for path in self._paths:
       if os.path.exists(path):
-        self.state.output.append((os.path.basename(path), path))
+        self.state.StoreContainer(containers.File(
```

Comment:
Please separate the creation of the container from storing it, to make this easier to read and debug